tests/test_linked_list.py

```diff
@@ -0,0 +1,112 @@
+"""Test the linked_list implementation."""
+import pytest
+from linked_list import LinkedList
+
+PARAMS_TABLE = [
+    [1],
+    [1, 2],
+    [1, 2, 3],
+    list(range(20)),
+    list(range(40)),
+    list(range(3)),
+    list(range(10))
+]
+
+
+def test_linked_list_head():
+    """Tests that a new LinkedList has a head"""
+    new_list = LinkedList()
+    assert hasattr(new_list, "head")
```

Comment:
Personally, I would make `test_linked_list_head` and `test_linked_list_head_is_none` into one test -- something like `test_empty_linked_list`. But that's pure personal style, and there is nothing wrong with what you have.